utilities/rpp-performancetests/HOST_NEW/Tensor_host_pkd3.cpp

```diff
@@ -1265,6 +1268,47 @@ int main(int argc, char **argv)
 
             break;
         }
+        case 35:
+        {
+            test_case_name = "glitch";
+            Rpp32u x_offset_r[images];
```

Comment:
Please change variables to camel case
Applicable all such instances for glitch in Unit tests and performance tests for HOST and HIP